cads_adaptors/constraints.py

```diff
@@ -684,7 +684,8 @@ def legacy_intersect_constraints(
     if len(requests) == 0:
         context.add_user_visible_error(
             "Your request has not produce a valid combination of values, please check your selection.\n"
-            "If using the cdsapi, please ensure that all values are strings or lists of strings.\n"
+            "If using the cdsapi, please ensure that the values in your request match the values provided"
+            f" in the web-portal, your request:\n {request}"
```

Comment:
Maybe a "\n" after "{request}", unless you believe there is one already in request.

Reply:
addresses in ca67d16